pkg/log/accesslog_test.go

```diff
@@ -654,25 +685,25 @@ func upstreamHostGetter(ctx context.Context, value *variable.IndexedValue, data
 }
 
 func requestHeaderMapGetter(ctx context.Context, value *variable.IndexedValue, data interface{}) (string, error) {
-	headers := ctx.Value(requestHeaderMapKey).(api.HeaderMap)
+	headers := ctx.Value(requestHeaderMapKey).(map[string]string)
```

Comment:
Maybe it would be better to modify `prepareLocalIpv4Ctx()` and `prepareLocalIpv6Ctx`.